Storybook docs (#5552)
bitwarden/clients: libs/components/src/tabs/tabs.mdx

import { Meta, Story, Primary, Controls } from "@storybook/addon-docs";

import * as stories from "./tabs.stories";

<Meta of={stories} />

# Tabs

The tab navigation and content tabs share the same styling. The tab navigation uses links to
navigate between pages, whereas the tab list uses `<buttons>` to toggle content on a single page.

Tabs should be displayed on the `background-alt` color, with their content area set to background
and 1rem of padding on the left and right.

~20px of padding is used bellow the tabs and before the section content.

> For focus states, use `focus-visible`.

Reviewer: I think we can remove this.

<Primary />
<Controls />

## Content Tabs

<Story of={stories.ContentTabs} />

## Navigation Tabs

<Story of={stories.NavigationTabs} />

## Content tabs in dialogs

Tabs can be used in dialogs to separate related content.

<Story id="component-library-dialogs-dialog--tab-content" />

## Accessibility

> **Navigation tabs** should be implemented using the `<nav>` element and `<a>` for each tab.

Reviewer: ```suggestion
**Navigation tabs** are implemented using the `<nav>` element and `<a>` for each tab.
```
